src/test/java/fr/quatrevieux/araknemu/game/fight/ai/simulation/effect/SetStateSimulatorTest.java

```diff
@@ -54,6 +58,7 @@ public void setUp() throws Exception {
             .state(50, -500)
             .state(1, 50)
         ;
+        ai = new FighterAI(fighter, fight, new NullGenerator());
```

Comment:
PlayerFighter is used so maybe should also use AIFighter and not FighterAI.

Reply:
PlayerFighter is implementation of Fighter for Player;
FighterAI is implentation of AI for Fighter instance.
AIFighter will means implementation of Fighter with an AI, which is not the case here.